backend/src/API/teamAPI.ts

```diff
@@ -106,6 +131,12 @@ export const setTeam = async (teamBody: Team, member: IdolMember): Promise<Team>
   return teamBody;
 };
 
+/**
+ * Deletes a team by removing all members, leaders, and formerMembers.
+ * @param teamBody - The Team object that will be deleted.
+ * @param member - The IdolMember that is requesting to delete the team.
+ * @returns A promise that resolves to the deleted Team object.
+ */
 export const deleteTeam = async (teamBody: Team, member: IdolMember): Promise<Team> => {
```

Comment:
This can be in a separate PR, but I think this function should be renamed to `clearTeam`, since the team document doesn't actually get deleted from the DB. Assuming that is intentional, all we're doing here is removing all members, leaders, and formerMembers from the team.